blocks/subscriptions-block/intl.json

```diff
@@ -0,0 +1,2939 @@
+{
```

Comment:
These need to be added to the appropriate files in `~/locale` as well. This document https://arcpublishing.atlassian.net/wiki/spaces/TI/pages/2538275032/Arc+Themes+Blocks+Internationalisation shows the steps for adding new keys properly. I can help with the actual uploading to Lokalise since you won't have access to our project.